src/fenicsxconcrete/finite_element_problem/linear_elasticity.py

```diff
@@ -72,21 +81,22 @@ def default_parameters() -> tuple[Experiment, dict[str, pint.Quantity]]:
         experiment = CantileverBeam(CantileverBeam.default_parameters())
 
         model_parameters = {}
-        model_parameters['rho'] = 7750 * ureg('kg/m^3')
-        model_parameters['E'] = 210e9 * ureg('N/m^2')
-        model_parameters['nu'] = 0.28 * ureg('')
+        model_parameters["rho"] = 7750 * ureg("kg/m^3")
+        model_parameters["E"] = 210e9 * ureg("N/m^2")
+        model_parameters["nu"] = 0.28 * ureg("")
```

Comment:
again, this has nothing to do with typing.
we need to decide whether single or double quotes should be used and stick to that.

Reply:
see above